### PR TITLE
Let Explorer Service provide date/time for next test run

### LINKED ISSUE #11: Explorer: Read next date from config.properties and update UI
We need to be able to change the next date in production, without re-deploying the Explorer. Current hardcoded solutions is hard to maintain in the long run.

### PR DESCRIPTION
**Pull request by laszlourszuly**
Fixes: #11

### CODE REVIEW

File: explorer/app2/index.html [hunk 1]
```diff
@@ -59,7 +59,7 @@
                 <div class="row">
                     <div class="col">
                         <div class="counter-header1" style="color: #32354E">Next test run will start at</div>
-                        <div id="next-date-hardcoded" class="counter-header2" style="color: #32354E">13 June, 13:30 (BST) Time in London, UK</div>
```
Comment (blenessy) [inline explorer/app2/index.html:62]: Considering that the next date on line 186 is different, you are using the same class attribute (next-datetime), which makes me assume that the rendered text will be the **same**. Is this intentional ?

**Review by blenessy — APPROVED** [review]
LGTM, just have a question.

File: explorer/app2/index.html (hunk 1 above)
Comment (laszlourszuly) [reply explorer/app2/index.html:62]: Glad that you noticed!

How should we handle "page 0" (`"state": "UNKNOWN"` in the metrics JSON)? Should it be the owner of the `"testNext"` meta attribute in the metrics JSON? Should we then introduce a `"testNextNext"` attribute for the summary page? Right now `"testNext"` means the test that comes after the one that's about to be executed or is currently executing.

Comment (laszlourszuly) [reply explorer/app2/index.html:62]: This means, of course, that "page 0" will show the wrong date/time as is now.

**Pull request merged**